lib/dl_file_uploader_worker_lib/dl_file_uploader_worker_lib/tasks/download_yadocs.py

```diff
@@ -88,6 +89,10 @@ async def run(self) -> TaskResult:
                     return Success()
 
             dfile.filename = spreadsheet_meta["name"]
+            xlsx_suffix = ".xlsx"
+            if not dfile.filename.endswith(xlsx_suffix):
+                raise exc.UnsupportedDocument
```

Comment:
Let's save the exact reason the file was rejected in the error details (in future PRs)
Something like this
```py
raise exc.UnsupportedDocument(details={"reason": f"Supported file extensions are: '.xlsx'. Got {dfile.filename}"})
```